site/components/PodcastCard/styles.ts

```diff
@@ -25,6 +25,17 @@ export const content = css`
   display: flex;
   flex-direction: column;
   gap: 1.2rem;
+
+  svg {
+    width: 40px;
+    height: 40px;
+    cursor: pointer;
```

Comment:
always use rem so that all elements scale along with the users browser font size
```suggestion
    width: 4rem;
    height: 4rem;
    cursor: pointer;
```

Reply:
Apologies my bad, I took the pixel value from the figma file and forgot to convert - done.